a3p-integration/proposals/p:upgrade-19/eval.sh

```diff
@@ -1,3 +1,11 @@
 #!/bin/bash
 
-# Do nothing because this is only testing the state resulting from previous proposals.
+# add gov4 address
+
+echo "UPGRADE-19  Running proposal declared in package.json"
+# copy to run in the proposal package so the dependencies can be resolved
+cp /usr/src/upgrade-test-scripts/eval_submission.js .
+
+#echo RUNNING  ./eval_submission.js
+#./eval_submission.js
```

Comment:
Is execution of `./eval_submission.js` commented out on purpose?

Reply:
Nope. I cleaned up other parts of the file while I was here.